setup.py

```diff
@@ -18,5 +18,10 @@
 	name="aclpubcheck",
 	install_requires=install_requires,
 	version="0.1",
-	scripts=[]
+	scripts=[],
+	entry_points = {
+		'console_scripts': [
+			"aclpubcheck=aclpubcheck.formatchecker:main",
```

Comment:
Would it be a good idea to rename `formatchecker.py` to `__main__.py` and change the console script to be:

```
"aclpubcheck=aclpubcheck.__main__:main"
```

This would allow parity with a `-m` style CLI call, i.e.:

```
python3 -m aclpubcheck --help
```

would be equivalent to:
```
aclpubcheck --help
```

This would mostly be useful in environments where the script shim is not (correctly/allowed to be) generated.

Reply:
@tnaumann Thanks. I cannot push to a different fork, so I made a similar PR instead: https://github.com/acl-org/aclpubcheck/pull/66